components/selection/selection-input.js

```diff
@@ -92,15 +99,20 @@ class Input extends SkeletonMixin(LabelledMixin(LitElement)) {
 	render() {
 		if (!this._provider) return;
 		if (this._provider.selectionSingle) {
+
 			const radioClasses = {
 				'd2l-input-radio': true,
 				'd2l-selection-input-radio': true,
 				'd2l-skeletize': true,
 				'd2l-hovering': this.hovering,
 				'd2l-disabled': this.disabled
 			};
+
+			const description = this.description ? html`<div id="${this._descriptionId}" hidden>${this.description}</div>` : null;
```

Comment:
In general I think we prefer `nothing` in these scenarios?

```suggestion
			const description = this.description ? html`<div id="${this._descriptionId}" hidden>${this.description}</div>` : nothing;
```